empress/topoSort/ReconBuilder.py

```diff
@@ -0,0 +1,277 @@
+# ReconBuilder.py
```

Comment:
Do you want me to rename `topoSort` to `topo_sort` and `ReconBuilder` to `recon_builder` for you?

Reply:
I can do it, thanks for suggesting